scss/tech.scss

```diff
@@ -0,0 +1,56 @@
+.techs {
+  display: block;
+  margin: 0 auto;
+  max-width: 1000px;
+
+  .tech {
+    display: inline-block;
```

Comment:
I'd use display flex in the parent to align the tchs in a row and display flex here to align the content in a column